packages/block-library/src/cover/edit.js

```diff
@@ -458,14 +493,24 @@ function CoverEdit( {
 				/>
 			</BlockControls>
 			<BlockControls group="other">
-				<MediaReplaceFlow
-					mediaId={ id }
-					mediaURL={ url }
-					allowedTypes={ ALLOWED_MEDIA_TYPES }
-					accept="image/*,video/*"
-					onSelect={ onSelectMedia }
-					name={ ! url ? __( 'Add Media' ) : __( 'Replace' ) }
-				/>
+				{ !! postId && (
+					<ToolbarButton
+						icon={ postFeaturedImage }
+						label={ __( 'Use featured image' ) }
+						isPressed={ useFeaturedImage }
+						onClick={ toggleUseFeaturedImage }
+					/>
+				) }
```

Comment:
```suggestion
				
					<ToolbarButton
						icon={ postFeaturedImage }
						label={ __( 'Use featured image' ) }
						isPressed={ useFeaturedImage }
						onClick={ toggleUseFeaturedImage }
					/>
				
```

@scruffian and I were just testing this PR on a pairing session and we've noticed that if you're editing a template which does not provide the context of a post then you cannot set to use feature image.

Why is this important? 

Try editing the template `single-post.html`. This doesn't supply "post" context but you might still want to user a cover block which uses the feature image.

If we remove this guard then it will enable this important functionality. AFAIK it won't have any downsides.